html/elements/input/password.json

```diff
@@ -0,0 +1,1070 @@
+                },
+                "edge": {
+                  "version_added": null
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": "1"
+                },
+                "firefox_android": {
+                  "version_added": "4"
+                },
+                "ie": {
+                  "version_added": "2"
+                },
+                "opera": {
+                  "version_added": "2"
+                },
+                "opera_android": {
+                  "version_added": true
+                },
+                "safari": {
+                  "version_added": "1"
+                },
+                "safari_ios": {
+                  "version_added": true
+                },
+                "webview_android": {
+                  "version_added": null
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "crossed-lock": {
+            "__compat": {
+              "description": "Crossed out lock in address bar to indicate insecure login page",
+              "support": {
+                "chrome": {
+                  "version_added": null,
+                  "notes": "implementing something similar"
+                },
+                "chrome_android": {
+                  "version_added": null,
+                  "notes": "implementing something similar"
+                },
+                "edge": {
+                  "version_added": null
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": "51"
+                },
+                "firefox_android": {
+                  "version_added": "51"
+                },
+                "ie": {
+                  "version_added": null
+                },
+                "opera": {
+                  "version_added": null
+                },
+                "opera_android": {
+                  "version_added": null
+                },
+                "safari": {
+                  "version_added": null
+                },
+                "safari_ios": {
+                  "version_added": null
+                },
+                "webview_android": {
+                  "version_added": null
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "insecure-login": {
+            "__compat": {
+              "description": "Message displayed next to password field to indicate insecure login page, plus autofill disabled",
```

Comment:
This seems like putting documentation in the description field. This does seem more relevant to password than "crossed-lock", but perhaps it would be better for the description to be kept short and there to be text in the main doc fleshing it out? Like "Special handling of password inputs in insecure login pages"?

Not much shorter :) but I think it is better for the MDN page body to include the description of exactly what the browser does in this situation.